R/trt_effects.R

```diff
@@ -136,11 +136,17 @@ effect_draw <- function(x, n, transform=NULL) {
 effect_plot <- function(..., transform=NULL) {
   l <- list(...)
 
-  caption <- "Possible treatment effect"
+  caption <- list(
+    title = "Estimate of pooled treatment effect",
```

Comment:
I like "possible" because if it's "estimate" people will assume it's the mean. And it's a combination of mean and dispersion that produces `effect_`, so please keep the old name or propose something suggestive of the fact.

Reply:
@wwiecek Maybe "Posterior distribution for pooled treatment effect"?

Reply:
If you say "pooled TE", people will understand it as a mean, I think, without thinking about across-groups variability.